api/test.sh

```diff
@@ -1,7 +1,9 @@
 if [ -z "$CI" ]; then
-    createdb -p 5433 scoreboard_tests && 
-    ava tests/*.test.js ; 
-    dropdb  -p 5433 scoreboard_tests
+    docker-compose exec db createdb -U postgres scoreboard_tests
```

Comment:
I had a hard time getting tests to run and made these changes.

Not sure if there are any issues here, but this is working well locally.

I need to revise the `else` part of this statement to work on CI.